### ISSUE TITLE
Function arguments

### ISSUE THREAD

**Issue by Sighmir** [B3nnyL/react-zendesk#13]
Hello, because of the type of ZendeskAPI expecting a string[] as arguments I have to use something like this for function arguments:
```ts
        ZendeskAPI('webWidget:on', 'chat:end', (() => {
          console.log('successfully ended a Zendesk Chat session!');
        }) as any);
```
Maybe this could be solved by setting the type to  (...args: any[]) => {} or  (...args: (string | function)[]) => {} ?

I don't like using any much.. but this way it would support whatever new arguments the API implements without needing changes, and the API would document all the arguments so we would have a good documentation about the types.

**Comment by gundam0910** [comment]
this fix my needs

**Comment by B3nnyL** [comment]
Sorry for late fix, it fixed by #18
